### ISSUE TITLE
Fails to deploy

### ISSUE THREAD

**Issue by Viper-7** [yudai/cf_nise_installer#10]
Cannot get either the standard or with-services versions to deploy on a clean Ubuntu x64 10.04 VM.

It looks like there have been changes to repositories linked to cf-release, such as gorouter, which are not included in the services snapshot, and are no longer compatible with your manifest.

Seems related to the p("x") helpers, job 'micro' fails, they added more? :/
http://grokbase.com/p/cloudfoundry.org/vcap-dev/136bfq5zfv/script-which-jobs-using-new-p-x-y-z-helpers-and-which-arent

Quoted from Dr Nic from that thread:
Doh, sorry this seems to give the correct (shorter) list:

$ grep '^properties:' jobs/*/spec | awk '{print $1}' | cut -d / -f 2 | uniq
cloud_controller_ng
gorouter

To summarize: these two jobs can be colocated; but I think not with any
other job today. Or something like that.

My current results:
grep '^properties:' jobs/*/spec | awk '{print $1}' | cut -d / -f 2 | uniq
cloud_controller_ng
collector
dea_next
gorouter
health_manager_next
login
nats
saml_login
syslog_aggregator
uaa
